api/v1/strict_server.go

```diff
@@ -337,15 +353,27 @@ func (srv *StrictServerImpl) GetRuntimeEvents(ctx context.Context, request apiTy
 }
 
 func (srv *StrictServerImpl) GetRuntimeAccountsAddress(ctx context.Context, request apiTypes.GetRuntimeAccountsAddressRequestObject) (apiTypes.GetRuntimeAccountsAddressResponseObject, error) {
-	account, err := srv.dbClient.RuntimeAccount(ctx, request.Address)
+	ocAddr, err := apiTypes.UnmarshalToOcAddress(&request.Address)
+	if err != nil {
+		return nil, err
+	}
+	account, err := srv.dbClient.RuntimeAccount(ctx, *ocAddr)
```

Comment:
`UnmarshalToOcAddress` can return `nil, nil`, but here we dereference `ocAddr`

Reply:
Hrm technically it should never happen because the `request.Address` is required to be non-nil. Added an explicit check+comment anyway for maintainability though.